src/cobra/core/compartment.py

```diff
@@ -64,69 +44,32 @@ def __init__(
         members : iterable, optional
             A DictList containing references to cobra.Model-associated objects
             that belong to the group.
-        kind : {"collection", "classification", "partonomy"}, optional
-            The kind of group, as specified for the Groups feature in the SBML
-            level 3 package specification.
-        """
-        Object.__init__(self, id, name)
+        dimensions: float, optional
```

Comment:
I was curious in what cases are non-integer dimensions used, do you know?

Reply:
For some models fractional dimensions exist, often fractal dimensions.
This can occur for instance in the context of vessel geometries/trees where you can have effective geometries between 2 and 3. I am sure there are other examples. I would say for almost every application in constraint based modeling this will be an integer. For compatibility with the SBML standard we could leave this a float, otherwise we have to cast this in reading/writing.